gen/gen.go

```diff
@@ -119,23 +146,69 @@ func Gen(exercise string, j interface{}, t *template.Template) error {
 	return ioutil.WriteFile(filepath.Join(dirExercise, "cases_test.go"), src, 0666)
 }
 
-func getPath(jFile string) (jPath, jOrigin, jCommit string) {
+func getLocal(jFile string) (jPath, jOrigin, jCommit string) {
 	// Ideally draw from a .json which is pulled from the official x-common
 	// repository.  For development however, accept a file in current directory
 	// if there is no .json in source control.  Also allow an override in any
 	// case by environment variable.
-	if jPath = os.Getenv("EXTEST"); jPath > "" {
+	if jPath := os.Getenv("EXTEST"); jPath > "" {
```

Comment:
If the caller has set EXTEST in env, but it (accidently) references a nonexistent file it will then keep rolling on to the remote. It does log which is helpful so you can see that the directory EXTEST references is no good as a data source. Should it be rolling on to the remote at that point or fail loud? I would think if EXTEST is defined than there was some intention of using it, in which case not having that file present may be worth shouting about?